weblate/static/js/vendor/main.js.license

```diff
@@ -0,0 +1,6 @@
+Copyright © Michal Čihař <michal@weblate.org>
+
+SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
This is definitely wrong, it should list all applicable licenses:

```suggestion
SPDX-License-Identifier: GPL-3.0-or-later AND MIT
```

Or we might choose to license our JavaScript under MIT to make it a bit simpler.

Reply:
Alright if MIT license is okay we will go with it

Reply:
I think it's okay to use for new stub files like this one. For existing code, we would have to ask the current authors, what could be a lengthy process which I'd like to avoid for now.